pkgs/by-name/li/livekit/package.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "livekit";
+  version = "1.5.1";
+
+  src = fetchFromGitHub {
+    owner = "livekit";
+    repo = "livekit";
+    rev = "v${version}";
+    hash = "sha256-3KRES/2mGO6b1ZZEGx29Yu5wgEG4NOJ7/J0xPvQiNWk=";
+  };
+
+  vendorHash = "sha256-5wByIkMs3321u4/2vPpsZ/L5zlcgrZo0b+NjeMR1RWE=";
+
+  subPackages = [ "cmd/server" ];
+
+  postInstall = "mv $out/bin/server $out/bin/livekit-server";
```

Comment:
```suggestion
  postInstall = ''
    mv $out/bin/server $out/bin/livekit-server
  '';
```

Reply:
Updated! PTAL. Is there any place where these kind of code conventions are documented?

Reply:
I can't find any references, but for maintainability, it would be better to use an indented string for shell script.